conda/README.md

```diff
@@ -8,3 +8,94 @@ This custom ML Runtime example demonstrates how to add the Conda package manager
 3. Add the runtime to CDSW/CML
 4. From a Project, open a new Python 3.9 session and select this ML runtime
 
+A pre-built version of the JupyterLab Conda runtime is published to:
+`docker.io/peterableda/conda-runtime:5`
+
+
+
+# JupyterLab Conda Runtimes
+The Conda ML Runtime is configured with the conda-forge channel by default.
+```
+cdsw@ry8tgfl7c2ncx03s:~$ conda config --show channels
+channels:
+  - conda-forge
+```
+
+## Example setup
+You can create new conda environments with the conda create command:
```

Comment:
Maybe we should mention that this command needs to be executed within a CML Session?